### PR TITLE
Release v1.52.0

### PR DESCRIPTION
**Pull request by albertteoh**

### CODE REVIEW

File: jaeger-ui
Comment (albertteoh) [inline jaeger-ui:1]: - [ ]  @albertteoh: update submodule once https://github.com/jaegertracing/jaeger-ui/pull/2020 is merged and tagged.

File: RELEASE.md [hunk 1]
```diff
@@ -54,8 +54,8 @@ Here are the release managers for future versions with the tentative release dat
 
 | Version | Release Manager | Tentative release date |
 |---------|-----------------|------------------------|
-| 1.52.0  | @jkowall        | 5 December 2023        |
 | 1.53.0  | @pavolloffay    | 3 January 2024         |
 | 1.54.0  | @joe-elliott    | 7 February 2024        |
-| 1.55.0  | @albertteoh     | 6 March 2024           |
+| 1.55.0  | @jkowall        | 6 March 2024           |
```
Comment (albertteoh) [inline RELEASE.md:59]: @jkowall and I are swapping shifts 😄

**Review by yurishkuro — APPROVED** [review]

**Review by yurishkuro — APPROVED** [review]

**Comment by albertteoh** [comment]
Hmmm... [tests are failing](https://github.com/jaegertracing/jaeger/actions/runs/7109023965/job/19356117205?pr=4991#step:12:619); I can take a look at this later tonight, unless if you know off the top of your head @yurishkuro?:

```
=== RUN   TestAllInOne/checkWebUI/logo
    all_in_one_test.go:95: 
        	Error Trace:	/home/runner/work/jaeger/jaeger/cmd/all-in-one/all_in_one_test.go:95
        	Error:      	Not equal: 
        	            	expected: 200
        	            	actual  : 404
        	Test:       	TestAllInOne/checkWebUI/logo
```

**Comment by albertteoh** [comment]
> Hmmm... [tests are failing](https://github.com/jaegertracing/jaeger/actions/runs/7109023965/job/19356117205?pr=4991#step:12:619)

Reproduced the problem and confirmed the following URL is able to display the logo:

Screenshot 2023-12-06 at 8 32 39 pm

**Pull request merged**